.eslintrc.yaml

```diff
@@ -21,7 +21,9 @@ rules:
       singleValue: false
   eqeqeq: 0
   chai-friendly/no-unused-expressions: 2
-  comma-dangle: 0
+  comma-dangle:
+    - error
+    - always-multiline
```

Comment:
I approve and merge so that we do not get too many files conflict 

But I want to rediscuss `always-multiline` which I find pretty weird but fine